bears/general/KeywordBear.py

```diff
@@ -92,6 +97,9 @@ def run(self,
         '''
         comments = _get_comments(dependency_results)
 
+        if not comments:
+            return
```

Comment:
Well it is looking for them inspite of these statements.
![image](https://cloud.githubusercontent.com/assets/17270966/21761407/c6ba3a4e-d678-11e6-8e56-6b733d7bb532.png)